test/spec/form.spec.js

```diff
@@ -1724,6 +1724,77 @@ describe('Form.prototype', () => {
     });
 });
 
+describe('Missing external instances', () => {
+    it('does not produce extraneous/unrelated errors', () => {
+        const options = { full: true };
+        const session = {
```

Comment:
This could just be an empty session, right? As-is it looks like the session is meaningful.

Reply:
The session was actually meaningful for one of the errors this change eliminated. It’s been a while, so I can’t recall if (or why exactly) I had to populate the whole thing… it’s possible a smaller subset would work and I can check that tomorrow too. But it definitely didn’t reproduce the error with an empty session.